### PR TITLE
Fix missing meta:enum and mis-matching between enum and meta:enum

### PR DESCRIPTION
**Pull request by jwen-adobe**

### CODE REVIEW

File: extensions/adobe/experience/decisioning/profile-constraint-details.schema.json [hunk 1]
```diff
@@ -35,8 +35,8 @@
           "meta:enum": {
             "none": "",
             "eligibilityRule": "The profile constraint is expressed as a single rule that must evaluate to true before the constrained action is allowed",
-            "Any Segments": "The profile constraint is expressed as one or more segments and the profile must be a member of at least one of them before the constrained action is allowed",
-            "All Segments": "The profile constraint is expressed as one or more segments and the profile must be a member of all of them before the constrained action is allowed",
+            "anySegments": "The profile constraint is expressed as one or more segments and the profile must be a member of at least one of them before the constrained action is allowed",
+            "allSegments": "The profile constraint is expressed as one or more segments and the profile must be a member of all of them before the constrained action is allowed",
             "rules": "The profile constraint is expressed as a number of different rules, e.g. eligibility, applicability, suitability, which all must evaluate to true before the constrained action is allowed"
```
Comment (hanessian) [inline extensions/adobe/experience/decisioning/profile-constraint-details.schema.json:40]: These meta:enums seem odd- @mjasrotia / @prabhum2 - any idea where they came from so we can track down if this is the desired label on the meta:enum?

File: extensions/adobe/experience/decisioning/ranking-details.schema.json [hunk 2]
```diff
@@ -40,7 +40,12 @@
           "enum": ["static", "scoringFunction", "rankingStrategy"],
           "description": "Specifies which order evaluation mechanism is used, static priority of the decision options, a scoring function that calculates a  numeric value for every option or a ranking strategy that receives a list to order it.",
           "meta:titleId": "ranking-details##xdm:orderEvaluationType##title##8401",
-          "meta:descriptionId": "ranking-details##xdm:orderEvaluationType##description##94251"
+          "meta:descriptionId": "ranking-details##xdm:orderEvaluationType##description##94251",
+          "meta:enum": {
+            "static": "static",
+            "scoringFunction": "scoringFunction",
+            "rankingStrategy": "rankingStrategy"
+          }
```
Comment (hanessian) [inline extensions/adobe/experience/decisioning/ranking-details.schema.json:48]: Static, Scoring Function, Ranking Strategy

File: extensions/airship/airship-event.schema.json [hunk 3]
```diff
@@ -323,7 +323,11 @@
                     "commercial"
                   ],
                   "meta:titleId": "airship-event##airship:messageType##title##79981",
-                  "meta:descriptionId": "airship-event##airship:messageType##description##49551"
+                  "meta:descriptionId": "airship-event##airship:messageType##description##49551",
+                  "meta:enum": {
+                    "unsubscribe": "unsubscribe",
+                    "commercial": "commercial"
+                  }
```
Comment (hanessian) [inline extensions/airship/airship-event.schema.json:330]: Cap labels

File: extensions/airship/airship-event.schema.json [hunk 4]
```diff
@@ -391,7 +395,11 @@
                 "exit"
               ],
               "meta:titleId": "airship-event##airship:regionAction##title##19401",
-              "meta:descriptionId": "airship-event##airship:regionAction##description##32271"
+              "meta:descriptionId": "airship-event##airship:regionAction##description##32271",
+              "meta:enum": {
+                "enter": "enter",
+                "exit": "exit"
+              }
             },
```
Comment (hanessian) [inline extensions/airship/airship-event.schema.json:403]: Enter, Exit

File: schemas/hygiene/aep-hygiene-ops-record.schema.json [hunk 5]
```diff
@@ -29,7 +29,11 @@
           "type": "string",
           "enum": ["deleteIdentity", "updateField"],
           "meta:titleId": "aep-hygiene-ops##xdm:action##title##96551",
-          "meta:descriptionId": "aep-hygiene-ops##xdm:action##description##71821"
+          "meta:descriptionId": "aep-hygiene-ops##xdm:action##description##71821",
+          "meta:enum": {
+            "deleteIdentity": "deleteIdentity",
+            "updateField": "updateField"
+          }
```
Comment (hanessian) [inline schemas/hygiene/aep-hygiene-ops-record.schema.json:36]: Delete Identity, Update Field

File: schemas/insights-metrics/dim-segment.schema.json [hunk 6]
```diff
@@ -54,7 +54,14 @@
...
               "description": "Current status of segment from external system normalized to active, inactive, deleted, draft or revoked.",
-              "enum": ["ACTIVE", "INACTIVE", "DELETED", "DRAFT", "REVOKED"]
+              "enum": ["ACTIVE", "INACTIVE", "DELETED", "DRAFT", "REVOKED"],
+              "meta:enum": {
+                "ACTIVE": "ACTIVE",
+                "INACTIVE": "INACTIVE",
+                "DELETED": "DELETED",
+                "DRAFT": "DRAFT",
+                "REVOKED": "REVOKED"
+              }
```
Comment (hanessian) [inline schemas/insights-metrics/dim-segment.schema.json:64]: ACTIVE: Active
INACTIVE, Inactive
etc

File: components/datatypes/deprecated/deprecated-consentpreferences.schema.json [hunk 7]
```diff
@@ -338,7 +338,7 @@
                 "iot_messages",
                 "social_media",
                 "other",
-                "none",
+                "no_preferred",
```
[the remark this answers is not in the document]
Comment (jwen-adobe) [reply components/datatypes/deprecated/deprecated-consentpreferences.schema.json]: i fixed this although it is deprecated.

**Comment by jwen-adobe** [comment]
@hanessian All fixed.

File: extensions/adobe/experience/adcloud/searchadvertising/campaign.schema.json [hunk 8]
```diff
@@ -111,11 +111,11 @@
             "Day of Week"
           ],
           "meta:enum": {
-            "daily": "Daily",
+            "Daily": "Daily",
             "Monthly": "Monthly",
             "Lifetime": "Lifetime",
             "Daily Spend Until Depleted": "Daily spend until depleted",
-            "Day of week": "Day of week"
+            "Day of Week": "Day of week"
```
Comment (hanessian) [inline extensions/adobe/experience/adcloud/searchadvertising/campaign.schema.json:118]: Day of Week

File: extensions/adobe/experience/decisioning/profile-constraint-details.schema.json (hunk 1 above)
Comment (hanessian) [reply extensions/adobe/experience/decisioning/profile-constraint-details.schema.json:40]: still need confirmation from Manik or Prabhakar on this one

**Review by mjasrotia — APPROVED** [review]

**Review by hanessian — APPROVED** [review]

**Pull request merged**